doc/api.md

```diff
@@ -35,122 +93,73 @@ Attribute | Type | Note
-`offset` | an integer >= 0
+`size` | an integer >= 0
+`from` | an integer >= 0
+`limit` | alias for `size`
+`offset` | alias for `from`
+`status` | one of `pending`, `cancelled`, `complete`
+`start_date` | timestamp string
+`end_date` | timestamp.string
+`text` | full-text search string. See https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#query-string-syntax
+
+#### Example
+
+```bash
+% curl -H 'Authorization: Bearer your-auth-token' \
+  https://cap.18f.gov/api/v2/proposals?size=5&from=10&text=foo
+```
 
-##### Example
+### `GET /api/v2/proposals/:id`
 
-https://c2-dev.cf.18f.us/api/v1/users.json?limit=5&offset=10
+Fetch a specific [Proposal](#proposal).
 
-#### Response
+#### Query parameters
+
+None
 
-Returns an array of [Users](#user).
+#### Example
 
-```javascript
-[
-  {
-    "created_at": "2015-01-10T07:05:42.445Z",
-    "email_address": "liono@some-dot-gov.gov",
-    "first_name": "Liono",
-    "id": 43,
-    "last_name": "Appolo",
-    "updated_at": "2015-01-10T07:05:42.445Z"
-  },
-  // ...
-]
+```bash
+% curl -H 'Authorization: Bearer your-auth-token' \
+  https://cap.18f.gov/api/v2/proposals/12345
 ```
 
-### `GET /api/v1/ncr/work_orders.json`
+### `POST /api/v2/proposals`
 
-#### Query parameters
+Create a new [Proposal](#proposal). The root key is the client model slug, e.g. `ncr_work_order`.
 
-All are optional.
+#### Example
 
-Name | Values
---- | ---
-`limit` | an integer >= 0
-`offset` | an integer >= 0
-
-##### Example
-
-https://c2-dev.cf.18f.us/api/v1/ncr/work_orders.json?limit=5&offset=10
-
-#### Response
-
-Returns an array of [Work Orders](#ncr-work-order), in descending order of creation.
-
-```javascript
-[
-  {
-    "amount": "1000.00",
-    "building_number": "DC0017ZZ ,WHITE HOUSE-WEST WING1600 PA AVE. NW",
-    "code": "ABC",
-    "description": "Existing paint is starting to crack.",
-    "emergency": false,
-    "expense_type": "BA80",
-    "id": 16,
-    "name": "Blue paint for the Blue Room",
-    "not_to_exceed": false,
-    "office": "P1121209 Security Management",
-    "proposal": {
-      "steps": [
-        {
-          "id": 92,
-          "status": "pending",
-          "user": {
-            "created_at": "2015-01-10T07:05:42.445Z",
-            "id": 43,
-            "updated_at": "2015-01-10T07:05:42.445Z"
-          }
-        }
-      ],
-      "created_at": "2015-02-21T07:05:42.445Z",
-      "id": 12,
-      "requester": {
-        "created_at": "2015-02-10T07:05:42.445Z",
-        "id": 71,
-        "updated_at": "2015-02-10T07:05:42.445Z"
-      },
-      "status": "pending",
-      "updated_at": "2015-03-28T01:13:33.564Z"
-    },
-    "rwa_number": "123456A",
-    "vendor": "ACME Corp"
-  },
-  // ...
-]
+```bash
+% curl -i -X POST -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer your-auth-token' \
+  --data @proposal.json \
+  https://cap.18f.gov/api/v2/proposals
+```
+
+where `proposal.json` looks like:
```

Comment:
sure or I was thinking for now we could link to the client data models so someone could see which attrs are required